inference-engine/src/inference_engine/ie_core.cpp

```diff
@@ -205,6 +206,7 @@ class Core::Impl : public ICore {
 
     std::unordered_set<std::string> opsetNames;
     std::vector<IExtensionPtr> extensions;
+    ngraph::frontend::FrontEndManager frontEndManager;
```

Comment:
why?

Reply:
This is for future implementation of 'ReadNetwork'